modules/installation-osp-kuryr-octavia-configuration.adoc

```diff
@@ -116,13 +116,20 @@ backend is ML2/OVS. There is no need for modifications if the backend is
 ML2/OVN.
 ====
 
-. To enforce network policies across Services, like when traffic goes through
+. (Optional, not needed for Red Hat OpenStack Platform 16+) To enforce
```

Comment:
Would I take this option with RHOSP 16+ for other reasons?

Reply:
Can probably delete 'optional' section if the note later on is correct.

Reply:
ok